Source/SharpNeedle/SonicTeam/PointCloud.cs

```diff
@@ -0,0 +1,92 @@
+﻿namespace SharpNeedle.SonicTeam;
+using BINA;
+
+[NeedleResource("st/pointcloud", @"\.pc(model|col)$")]
+public class PointCloud : BinaryResource
+{
+    public new static readonly uint Signature = BinaryHelper.MakeSignature<uint>("CPIC");
+    public uint FormatVersion { get; set; } = 2; // Either 1 or 2, the data is the same regardless
```

Comment:
Are you sure this is file version or whether it's a pcmodel pr pccol file? I'm currently unable to verify but due to the nature of the format it doesn't sound unlikely.

Reply:
It's definitely not a model/col identifier, as I just checked myself and some pcmodel files have it as 1, 2 seems to be the most common though.